src/main/java/com/cgi/ipsen5/Controller/LocationController.java

```diff
@@ -23,7 +26,22 @@ public ApiResponse<List<Location>> getAllLocations() {
 
     @GetMapping(value = "/{id}")
     public ApiResponse<Location> getById(@PathVariable UUID id) {
-        return new ApiResponse<>(this.locationDao.getLocationById(id), HttpStatus.OK);
+        return new ApiResponse<>(this.locationService.getLocationById(id), HttpStatus.OK);
+    }
+
+    //TODO: find solution for req params
```

Comment:
Is een DTO hier geen goede solution for?

Reply:
done